src/ShowPage/ShowPage.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import { BrowserRouter, Route, Link } from 'react-router-dom';
+import { getSingleMovie } from '../apiCalls'
+import './ShowPage.css';
+
```

Comment:
Let's ditch all these spaces the next time we get this file open.

Reply:
When you've got changes like this, it's best to just add another commit fixing it before merging :) That way you don't forget!